app/_src/kubernetes-ingress-controller/guides/services/tls.md

```diff
@@ -55,13 +118,13 @@ The Ingress API supports TLS termination using the `.spec.tls` field. To termina
               name: demo-example-com-cert
     ```
 
-2. Bind a `HTTPRoute` to the `Gateway`.
+2. Bind an `HTTPRoute` to the `Gateway`.
 
     ```yaml
     apiVersion: gateway.networking.k8s.io/{{ gwapi_version }}
     kind: HTTPRoute
     metadata:
-      name: demo-example 
+      name: demo-example
     spec:
       parentRefs:
       - name: example-gateway
```

Comment:
Can we add a `curl` command that will verify that the connectivity is achieved?

Something like:

```
curl -i -v --cacert example.com-tls.crt -H "Host: demo.example.com" https://$PROXY_IP/echo
```

(I for some reason couldn't make this work but have verified that `curl -i -v -k -H "Host: demo.example.com" https://$PROXY_IP/echo` works. This will need to be verified so that the provided CA actually works)

Reply:
Added instructions to both modes.